### PR TITLE
Sofiia added new personal class

### PR DESCRIPTION
**Pull request by 16so03**
Sofiia added new personal class

### CODE REVIEW

File: src/Utils/Sofiia_Hello.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package Utils;
+
+public class Sofiia_Hello {
+    //Hello
+    //I can type whatever
```
Comment (Zumret910) [inline src/Utils/Sofiia_Hello.java:5]: hello
nice to see you

Comment (16so03) [reply src/Utils/Sofiia_Hello.java:5]: Hello
Nice to see you too

**Review by 16so03 — COMMENTED** [review]
Everything is done

**Comment by Zumret910** [comment]
good

**Comment by aishakariz** [comment]
Looks great!!

**Pull request merged**